app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
```

Comment:
I mean, I know @patybastos didn't touched this point. 
But I believe it can be fixed now to avoid future questions.